command/e2etest/provider_dev_test.go

```diff
@@ -73,4 +73,12 @@ func TestProviderDevOverrides(t *testing.T) {
 	if got, want := stdout, `Provider development overrides are in effect`; !strings.Contains(got, want) {
 		t.Errorf("stdout doesn't include the warning about development overrides\nwant: %s\n%s", want, got)
 	}
+
+	stdout, _, err = tf.Run("plan")
```

Comment:
Seems `stderr` should be reset here in case of printing it in the `t.Fatalf()` below

```suggestion
	stdout, stderr, err = tf.Run("plan")
```

Reply:
Oh yes, good catch! :taco:  Although we aren't doing any assertions on the error output, we are printing it out and it would be a confusing diversion to be trying to debug the previous error output instead of the current error output.